### PR TITLE
data: AndrewLamichhane

### PR DESCRIPTION
**Pull request by Andrew-LC**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5652"></a>

### CODE REVIEW

File: data/AndrewLamichhane [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+     {
+      "name": "Twitter",
+      "url": "",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/andrew-lamichhane-34a397219/",
+      "icon": "FaLinkedIN"
```
Comment (khushalarora) [inline data/AndrewLamichhane:14]: Please replace it with  **' FaLinkedin '** :+1: 
```Suggestion
      "icon": "FaLinkedin"
```

File: data/AndrewLamichhane [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+     {
+      "name": "Github",
+      "url": "https://github.com/Andrew-LC",
+      "icon": "Github"
```
Comment (khushalarora) [inline data/AndrewLamichhane:19]: This is not a valid icon, you can checkout [ICONS](https://linkfree.eddiehub.io/icons) to use the correct **GitHub** Icon :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @Andrew-LC, thank you for contributing  :tada: 
Please rename your file and use your Username `Andrew-LC` instead of your name.
Also, add `.json` at the end.
It will look like  `data/Andrew-LC.json`

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey Andrew, you still need to rename your file.
 
As suggested by **khushalarora** your filename must be exact same to your GitHub username. So please update it to `Andrew-LC.json`

**Pull request closed without merging**